spec/automation/unit/builtin_method_validation/parse_automation_request_spec.rb

```diff
@@ -4,39 +4,39 @@
 
   it "for miq_provision request" do
     ws = MiqAeEngine.instantiate("#{inst}?request=vm_provision", user)
-    expect(ws.root).to have_attributes(
+    expect(ws.root.attributes).to include(
```

Comment:
Do you need the `.attributes`calls in this file?

Reply:
Yes, `ws.root` is a `MiqAeEngine::MiqAeObject` object and without `attributes` you get the error message:

`expected #<MiqAeEngine::MiqAeObject:0x0000..., but it does not respond to 'include?'`